consoles/sshVirtsh.pm

```diff
@@ -327,18 +364,18 @@ sub define_and_start {
     $chan->close();
 
     # shut down possibly running previous test (just to be sure) - ignore errors
-    $self->{ssh}->channel()->exec("virsh destroy " . $self->name);
-    $self->{ssh}->channel()->exec("virsh undefine " . $self->name);
+    $self->{ssh}->channel()->exec("virsh " . "$remote_vmm" . " destroy " . $self->name);
```

Comment:
just put the variable in the string - no need to concat if it's a string (while $self->name is a function call) and I find "$virsh $remote_vmm destroy " easier to read